src/EventStore.Core/LogV2/LogV2Sizer.cs

```diff
@@ -0,0 +1,7 @@
+﻿using EventStore.Core.LogAbstraction;
+
+namespace EventStore.Core.LogV2 {
+	public class LogV2Sizer : ISizer<string> {
+		public int GetSizeInBytes(string t) => 2 * t.Length;
```

Comment:
```suggestion
		public int GetInMemorySizeInBytes(string t) => 2 * t.Length;
```

Would `GetInMemorySizeInBytes` be a better name for this method?
Based on the IndexWriter's `_committedEvents` bounded cache's code, it appears to be an estimate for the amount of memory taken by the stream name.

Reply:
hmm, yes good point. I'll rename the class to reflect that rather than the method, since its a property of the implementation really rather than the interface

Reply:
@timothycoleman thanks!